core/src/main/java/com/klaytn/caver/contract/ContractDeployParam.java

```diff
@@ -0,0 +1,36 @@
+package com.klaytn.caver.contract;
+
+
+import org.web3j.abi.datatypes.Type;
+
+import java.util.List;
+
+public class ContractDeployParam {
```

Comment:
Which name do you think is better, ContractDeployParam or ContractDeployParams?

Reply:
I think ContractDeployParams is better. The SendOptions class is also prefixed with s.

Reply:
Sounds good :) Thank you for your opinion! Can you change the code? I will modify the common architecture class diagram to `ContractDeployParams`.

Reply:
I rename to ContractDeployParams.